src/command.c

```diff
@@ -118,7 +118,7 @@ static struct dmp_cmb_list_entry *dv_cmb_allocate(struct device *dev)
 
 	do {
 		cmb->logical = dma_alloc_coherent(dev, alloc_size,
-			&cmb->physical, GFP_KERNEL);
+					&cmb->physical, GFP_KERNEL);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket